package.json

```diff
@@ -117,32 +123,25 @@
     "menus": {
       "commandPalette": [
         {
-          "command": "metals.restartServer",
-          "when": "editorLangId == scala"
+          "command": "metals.restartServer"
```

Comment:
I don't think we need the when clause, since we might be in a txt file and still run the import build. Besides with more and more supported tools this might be harder to maintain.

Reply:
that's a concern that I have as well. I would prefer not to change it in this PR, but I'm open to discuss it further.

The reason why the scoping was introduced was due to confusing errors when running the commands without the extension being active.

I don't love this solution either, so let maybe open an issue for that?

Reply:
I agree with @gabro, let's move this discussion into a separate issue since it's not strictly related to the Gradle setting.

Reply:
Reverted, should we add the possibility at least for gradle files?  `resourceExtname == .gradle`